lib/seattleflu/id3c/cli/command/etl/redcap_det_swab_n_send.py

```diff
@@ -352,18 +352,11 @@ def specimen_barcode(record: Any) -> str:
         value = barcode
     )
 
-    if not record.get('collection_date'):
-        LOG.warning("Could not create Specimen Resource due to lack of collection date.")
-        return None, None
-
     # YYYY-MM-DD in REDCap
-    collected_time = record['collection_date']
+    collected_time = record['collection_date'] if record['collection_date'] else None
 
     # YYYY-MM-DD HH:MM:SS in REDCap
-    received_time = record['samp_process_date'].split()[0]
-    if not received_time:
-        LOG.warning("No sample process date found. Using collection date instead.")
-        received_time = collected_time
+    received_time = record['samp_process_date'].split()[0] if record['samp_process_date'] else None
```

Comment:
It looks like we're doing more than just changing the handling of `collection_date` here. Perhaps we can either split the commit or edit the current commit's language to reflect this.